### DISCUSSION TITLE
Question about "raise the dead"

### DISCUSSION THREAD

**Discussion by Jiffydude** [TEHE-Studios/ExpandedHelicopterEvents#137] [Q&A] [answered]
I noticed that there is a "raise the dead" function on the debug menu of EHE. I was just wondering if this is tied to any of the events or is just a test function?

Along with other reports, I myself have noticed that there are some oddities (whether intentional or not I'm unsure of), such as players reporting respawning zeds inside/around their bases that are either secluded or cleared out.

One common mod used around these reports is EHE, so I'm just looking to clarify some things to help track this down.

Unfortunately, I'm not versed in coding (yet), so I wouldn't know how to look through the files of EHE, so that's another reason I'm pestering you here.

**Comment by Chuckleberry-Finn** [comment, marked as answer]
Reviving zombies is a vanilla feature. The test function is for testing event attacks.

**Comment by discotimetraveler** [comment]
To elaborate on this, because I have the same questions... if we disable vanilla zombie respawns, does this mod still spawn it's own zombies? Thanks

**Comment by Chuckleberry-Finn** [comment]
Yes, upon crashes and super-weird events.

**Reply by discotimetraveler** [reply]
ok, thanks. I ultimately determined "my issue" related to the server being weird and not this mod.
